spec/core/Integer.Format.Spec.savi

```diff
@@ -49,3 +49,7 @@
     assert: "\(U8[0].format.bin.without_leading_zeros)" == "0b0"
     assert: "\(I16[36].format.bin.without_leading_zeros)" == "0b100100"
     assert: "\(I16[-36].format.bin.without_leading_zeros)" == "0b1111111111011100"
+
+  :it "can format integers as Unicode codepoints"
+    assert: "\('S'.format.unicode)" == "S"
+    assert: "\(U8[83].format.unicode)" == "S"
```

Comment:
Can you also add at least one test for a non-ASCII unicode character?

Reply:
can I use `\U` escape sequences in Savi?

Reply:
Yes, but only with a lowercase `\u` - followed by 4 hexadecimal digits